### ISSUE TITLE
Not all theories are supported

### ISSUE THREAD

**Issue by vvs-** [milpgame/milpgame#4]
Progress bar stops at 97% while trying to load demo0.mm. There seems to be an error printed on stdout.

**Comment by milpgame** [comment]
Yes, you are right Milpgame works only for the moment with set.mm.
